bin/deploy-cert-manager.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
```

Comment:
decrypt key.json.enc - before creating the secret


rm key.json after creating the secret